pkg/gds/admin_test.go

```diff
@@ -531,6 +529,7 @@ func (s *gdsTestSuite) TestListVASPs() {
 
 	// List VASPs with the specified status
 	request.path = "/v2/vasps?status=" + snippets[0].VerificationStatus
+	actual = &admin.ListVASPsReply{}
```

Comment:
Chased a really weird bug around for an hour because we were not resetting the target pointer for the JSON marshal in `doRequest`. I guess we just got lucky and it wasn't causing us issues until now.